index.md

```diff
@@ -3,9 +3,11 @@ layout: home
 title: The Identity Selector Software
 ---
 
-The Identity Selector Software (thiss.io) is an implementation of an identity selector (aka discovery
-service). Currently the implementation supports the SAML metadata discovery protocol aswell as the 
-[RA21.org](https://ra21.org) [recommended practices for discovery UX](https://groups.niso.org/apps/group_public/download.php/21376/NISO_RP-27-2019_RA21_Identity_Discovery_and_Persistence-public_comment.pdf).
+![Coalition for Seamless Access Logo](/assets/img/RA21-New-Blue.png)
```

Comment:
I added several images, but I'm not quite sure if I can just use standard markdown to render them on the site, or if I need to format them differently. Just let me know if what I put won't work.